### ISSUE TITLE
Cannot read properties of null (reading 'contentDocument')

### ISSUE THREAD

**Issue by isaackearl** [VemtoOrg/vemto2-issues#28]
**Error Message:**
Cannot read properties of null (reading 'contentDocument')

**Stack:**
```
TypeError: Cannot read properties of null (reading 'contentDocument')
    at c (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:1227:32529)
```

---

Vemto 2.0.4
darwin 23.3.0
Locale: en-US
App version: 2.0.4

I'm just navigating around in a pretty much completely new app.  I'm not sure what is causing this error.

**Comment by TiagoSilvaPereira** [comment]
@isaackearl could you please check if this bug still happening on 2.0.7?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
